### PR TITLE
Add documentation on Kubernetes Namespace extension

### LINKED ISSUE #438: Update document for application-scope namespace
Parent feature - https://github.com/project-radius/radius/issues/3740

[AB#5305](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/5305)

### PR DESCRIPTION
**Pull request by AaronCrawfis**
Partially addresses https://github.com/project-radius/radius/issues/3740
Closes radius-project/radius#438

### CODE REVIEW

File: docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md [hunk 1]
```diff
@@ -11,15 +11,29 @@ When deploying Radius resources to environments running on Kubernetes, the [cont
 
 <img src="kubernetes-mapping.png" alt="Diagram showing Radius resources being mapped to Kubernetes objects" width=600px />
 
+## Namespace mapping
+
+Radius environments allow you to specify the Kubernetes namespace where environment-scoped resources are generated. For example, if you set a Radius environment to use the namespace 'default' and deploy an environment-scoped Dapr Link, the backing dapr.io/Component will be deployed into the 'default' namespace.
+
```
Comment (sk593) [inline docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md:17]: Should 'default', 'myapp', etc be coded as `default`, `myapp`, etc? They're showing up as text and not code on the staging site

**Review by sk593 — DISMISSED** [review]
I think the names of resources, namespaces, etc should be code instead of just text but other than that, LGTM!

File: docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md [hunk 2]
```diff
@@ -11,15 +11,29 @@ When deploying Radius resources to environments running on Kubernetes, the [cont
...
+Application-scoped resources are by default generated in a new Kubernetes namespace with the name format '<envNamespace>-<appname>'. For example, an application named 'myapp' with a container named 'frontend', deployed into the environment from above, will cause a Deployment named 'frontend' to be deployed into the namespace 'default-myapp'. This prevents multiple applications from conflicting with each other.
+
+If you would like to override the default behavior and specify your own namespace for application resources to be generated into, you can leverage the kubernetesNamespace container extension. All application-scoped resources will be deployed into this namespace instead.
```
Comment (youngbupark) [inline docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md]:
```suggestion
If you would like to override the default behavior and specify your own namespace for application resources to be generated into, you can leverage the `kubernetesNamespace` application extension. All application-scoped resources will be deployed into this namespace instead.
```

File: docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md [hunk 3]
```diff
@@ -11,15 +11,29 @@ When deploying Radius resources to environments running on Kubernetes, the [cont
 
 <img src="kubernetes-mapping.png" alt="Diagram showing Radius resources being mapped to Kubernetes objects" width=600px />
 
+## Namespace mapping
+
+Radius environments allow you to specify the Kubernetes namespace where environment-scoped resources are generated. For example, if you set a Radius environment to use the namespace 'default' and deploy an environment-scoped Dapr Link, the backing dapr.io/Component will be deployed into the 'default' namespace.
```
Comment (youngbupark) [inline docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md]: dapr.io/component is namespace-scoped resource. it means that it cannot be shared across multiple namespaces. it always need to be deployed to namespace where runtime is running. Also so far I do not see any example to support environment scope resource in LinkRP.

Comment (AaronCrawfis) [reply docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md]: Ah great point, environment-scoped Dapr resources are now a scenario that we no longer support by default. That's fine for now, we can have docs on how to handle this situation. Plus recipes are going to be the main experience for separation of concerns in the future anyway. Will remove this section

File: docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md (hunk 1 above)
Comment (AaronCrawfis) [reply docs/content/operations/platforms/kubernetes/kubernetes-mapping/index.md:17]: Updated, good catch!

**Pull request merged**